resources/text/units.md

```diff
@@ -0,0 +1,96 @@
+# Units
+
+Written because the author confused themselves and wanted to be sure they
+now understand.
+
+There are three main classes of coordinates in fonts:
+
+1. Design coordinates
+   * Used by the type designer and design tool
+   * Not directly present in the font binary at all
+   * For example, [<dimension xvalue/>](https://fonttools.readthedocs.io/en/latest/designspaceLib/xml.html#dimension) in a .designspace file
+1. User, aka External, coordainates
+   * The values in [fvar](https://learn.microsoft.com/en-us/typography/opentype/spec/fvar), for example Weight 400
+   * What you would expose to an end user in a document editor or manipulate via CSS
```

Comment:
the use of a numerical value to denote weight feels more like some ancient convention, and I don't really think of this in terms of 'coordinates', per se, but I can see how that makes sense. I'm now very curious to think about where this convention came from...

Reply:
@davelab6 do you happen to know where the weight scale comes from?